test/unconstrained.jl

```diff
@@ -103,8 +103,8 @@ alg4 = MethodOfSteps(DP5(), constrained=false, max_fixedpoint_iters=100,
 sol4 = solve(prob, alg4)
 
 @test abs(sol1[end] - sol2[end]) < 2.5e-11
-@test abs(sol1[end] - sol3[end]) < 1.3e-13
-@test abs(sol1[end] - sol4[end]) < 4.5e-13
+@test abs(sol1[end] - sol3[end]) < 1.3e-14
```

Comment:
why? won't this cause floating point issues?

Reply:
It didn't cause any issues on my computer and also not in the Travis builds (the failure there is due to a build error on Mac) and the Win32 build. Just two bounds in `residual_controls.jl` failed slightly, but in the range `1e-10` and `1e-11`, these here passed. I just wanted to make sure that any regressions or improvements are immediately visible in the tests. But I can just relax these bounds again.

Reply:
I would just relax them a bit. A regression which shows something actually broke will make this many orders of magnitude larger, and I think they'll be small enough that JuliaCIBot or other things can have random breakage due to just calculating floating points differently.